src/macros.rs

```diff
@@ -65,13 +67,23 @@ macro_rules! to_hex_string {
     }};
 }
 
+#[cfg(debug_assertions)]
+pub fn get_transaction_service_host(chain_info: ChainInfo) -> String {
+    return chain_info.transaction_service;
+}
+
+#[cfg(not(debug_assertions))]
+pub fn get_transaction_service_host(chain_info: ChainInfo) -> String {
+    return chain_info.vpc_transaction_service;
+}
```

Comment:
I would make these methods private. We don't need to expose them outside the macro context

Reply:
I think they need to be public to be accessible from the macro itself.

Reply:
Just confirmed and they do need to be public else `error[E0603]` is raised (`function `get_transaction_service_host` is private`)